Pinta.Tools/Tools/PaintBrushTool.cs

```diff
@@ -40,6 +41,8 @@ public sealed class PaintBrushTool : BaseBrushTool
 	private BasePaintBrush? default_brush;
 	private BasePaintBrush? active_brush;
 	private PointI? last_point = PointI.Zero;
+	private List<RepeatingDraw> open_repeating_draws = [];
```

Comment:
I found it confusing to figure out what was happening here - why is there only one Cairo.Context being saved, but there is a list of multiple draws?